command/rootchain/rootchain.go

```diff
@@ -66,6 +66,13 @@ func setFlags(cmd *cobra.Command) {
 		0,
 		"the payload type for the rootchain event. Possible types: [0 - ValidatorSetPayload]",
 	)
+
+	cmd.Flags().Uint64Var(
+		&params.blockConfirmations,
+		blockConfirmationsFlag,
+		0,
```

Comment:
I would put a default value as a bigger number (min 6)

Reply:
Why? It's `iota`

Reply:
I'm obviously blind, I thought you were talking about the payload type 🤦 

Bumped:
[1ac3b9e](https://github.com/0xPolygon/polygon-edge/pull/716/commits/1ac3b9e09d02d5bd5bd80941c99822368f166c24)